src/services/FlowChartServices.ts

```diff
@@ -62,6 +62,49 @@ export const sendApiKeyToDjango = async (apiKey: string) => {
     });
   }
 };
+export const sendS3KeyToDjango = async (
+  s3keyname: string,
+  s3AccessKey: string,
+  s3SecretKey: string
+) => {
+  try {
+    const response = await fetch(`${API_URI}/s3/set-s3-key`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        name: s3keyname,
+        accessKey: s3AccessKey,
+        secretKey: s3SecretKey,
+      }),
+    });
+
+    if (response.ok) {
+      const responseData = await response.json();
+      notifications.update({
+        id: "set-s3-key",
+        title: "Successful!",
+        message: "Successfully set the Access Key",
+        autoClose: 5000,
+      });
+    } else {
+      notifications.update({
```

Comment:
I think just throwing an error here would work to make it go to the catch block, makes for less repeating code.